internal/error.go

```diff
@@ -216,16 +217,24 @@ type (
 	}
 
 	temporalError struct {
+		messenger
 		originalFailure *failurepb.Failure
 	}
 
 	failureHolder interface {
 		setFailure(*failurepb.Failure)
 		failure() *failurepb.Failure
 	}
+
+	messenger interface {
+		message() string
+	}
```

Comment:
this is the breaking change we talked about few days ago?

Reply:
It was today morning :-) And actually not. This is just internal interface to simplify dealing with error messages.